ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -164,7 +181,7 @@ export default React.memo<Props>(function VideoJs(props: Props) {
         }
       };
     }
-  });
+  }, []); // Parameters are needed to avoid videojs to be loaded more than once (empty is allowed)
```

Comment:
Is there a main reason for that ? I mean, the problem is when videojs is rendered more than once there are also more than one instance of it in the DOM, this could lead to sending to chromecast the wrong instance of the player. Even though I didn't get any issue without the empty parameter, I do not feel comfortable when I see multiple instances of the same object. But that is maybe just me ^^

Reply:
It shouldn't create multiple instances because we are calling `player.dispose()` in the effect cleanup. If you add the empty array, a few things will stop working after you watch more than one video, because that effect won't get re-ran

Reply:
Understandable, so ok, if there are not multiple instances it should work also without the empty parameter